src/reportWebVitals.ts

```diff
@@ -1,8 +1,8 @@
 import { ReportHandler } from 'web-vitals';
 
 const reportWebVitals = (onPerfEntry?: ReportHandler) => {
-  if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+  if (onPerfEntry != null && onPerfEntry instanceof Function) {
+    void import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
```

Comment:
```suggestion
   import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
```
Ne pas utiliser void mais plutôt ajouter un catch

Reply:
J'ai supprimé le fichier finalement, car non utilisé.